runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -7294,17 +7298,120 @@ done:;
 	/*
 	 * Determine if the two valueTypes are substitutable when rhs.class equals lhs.class
 	 *
-	 * @param[in] lhs the lhs object of acmp bytecodes and it's a valueType
-	 * @param[in] rhs the rhs object of acmp bytecodes and it's a valueType
+	 * @param[in] lhs the lhs object address
+	 * @param[in] rhs the rhs object address
+	 * @param[in] startOffset the initial offset for the object
+	 * @param[in] clazz the value type class
 	 * return true if they are substitutable and false otherwise
 	 */
-	VMINLINE bool
-	isSubstitutable(j9object_t lhs, j9object_t rhs)
-	{
-		/*
-		 * TODO: this will be updated in a future PR.
-		 */
-		return false;
+	bool
+	isSubstitutable(j9object_t lhs, j9object_t rhs, UDATA startOffset, J9Class *clazz)
```

Comment:
please move this function to valueTypeHelpers.cpp and make it a static helper method